scripts/azure-linux_mac.yml

```diff
@@ -237,8 +236,10 @@ steps:
     # tiledb_unit is configured to set a job-level variable TILEDB_CI_SUCCESS=1
     # following the test run. If this variable is not set, the build should fail.
     # see https://github.com/TileDB-Inc/TileDB/pull/1400 (5f0623f4d3)
-    if [[ "$TILEDB_CI_SUCCESS" -ne 1 ]]; then
-      exit 1;
+    if [[ "$TILEDB_GCS" != "ON" ]]; then
```

Comment:
Please don't change this, it is a guardrail to avoid accidentally turning CI off without noticing (given some of azure's default behavior is broken, like not erroring out when a command fails).

Instead, use `echo "##vso[task.setvariable variable=TILEDB_CI_SUCCESS]1"` in the [GCS-specific block](https://github.com/TileDB-Inc/TileDB/pull/1585/files#diff-7139846aa3ce2c0e4831e229502279a9L159) where the tests will be run in the future.

see also
https://github.com/TileDB-Inc/TileDB/pull/1400/files#diff-5a475c942ea8e50ef80fe4a2373bef9bR16

Reply:
Done, thank you